Project.toml

```diff
@@ -0,0 +1,15 @@
+name = "AutoHashEquals"
+uuid = "97026771-8bcd-44fb-b7eb-f644370d63d3"
+authors = []
+version = "0.3.0"
```

Comment:
Might as well just make this 1.0 and drop the Julia versions from before 1, if we're making a "breaking" version change anyway.

Reply:
I was thinking the same. But I wasn't sure if we were ready to abandon the old versions for new features. (e.g. with a fix to #16, say.)

But yeah, I'm happy to do that if it sounds good to you! :)

Reply:
Done.